src/main/java/net/dries007/tfc/util/Helpers.java

```diff
@@ -368,6 +368,17 @@ public static void damageCraftingItem(ItemStack stack, int amount)
         }
     }
 
+    @Nonnull
+    public static ItemStack consumeItem(ItemStack stack, int amount)
+    {
+        if (stack.getCount() <= amount)
+        {
+            return ItemStack.EMPTY;
+        }
+        stack.shrink(amount);
+        return stack;
+    }
```

Comment:
- No need for `@Nonnull`
- This method shouldn't be necessary? the first bit is handled already by `shrink` and then all it's saving is `stack.shrink(amount); return stack` which seems a bit pedantic.